### PR TITLE
add note on list of supported software page + fixes for update-version-specific-docs.sh

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: docs/version-specific/update-version-specific-docs.sh [hunk 1]
```diff
@@ -147,17 +147,23 @@ echo "* [List of available toolchain options](toolchain-opts.md)" >> $overview
...
 echo "eb --list-toolchains"
-eb --list-toolchains --output-format md | sed 's/^\(# .*\)/\1 {: #toolchains_table }/g' > eb-list-toolchains.md
-echo >> eb-list-toolchains.md
-echo "*(see also* \`\`eb --list-toolchains\`\` *)*" >> eb-list-toolchains.md
+eb --list-toolchains --output-format md | sed 's/^\(# .*\)/\1 {: #toolchains_table }/g' > toolchains.md
+echo >> toolchains.md
+echo "*(see also* \`\`eb --list-toolchains\`\` *)*" >> toolchains.md
 
-echo "* [List of known toolchains](eb-list-toolchains.md)" >> $overview
+echo "* [List of known toolchains](toolchains.md)" >> $overview
```
Comment (boegel) [inline docs/version-specific/update-version-specific-docs.sh:154]: In #96 we renamed `eb-list-toolchains.md` to `toolchains.md` manually, but the script overwrites this again, so needs to be changed here too

File: docs/version-specific/supported-software.md [hunk 2]
```diff
@@ -27577,8 +27582,6 @@ version   |versionsuffix                |toolchain
 ``1.12.1``|                             |``foss/2021a``, ``foss/2022a``
 ``1.12.1``|``-CUDA-11.3.1``             |``foss/2021a``
 ``1.12.1``|``-CUDA-11.7.0``             |``foss/2022a``
-``1.13.1``|                             |``foss/2022a``
-``1.13.1``|``-CUDA-11.7.0``             |``foss/2022a``
```
Comment (boegel) [inline docs/version-specific/supported-software.md:27581]: Not sure how this got in here, probably local WIP easyconfigs - we don't have PyTorch 1.13.1 yet (we don't even have PRs for it yet)...

Comment (branfosj) [reply docs/version-specific/supported-software.md:27581]: Sorry. This happened when I was battling generating these pages. I was struggling to sort out a working system to generate this page on (see my comment in #90 about the number of changes I was getting at one point).

Comment (boegel) [reply docs/version-specific/supported-software.md:27581]: Ah, that clears it up. No worries :)

**Review by branfosj — APPROVED** [review]

**Pull request merged**